src/main/java/nl/novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -1,4 +1,51 @@
 package nl.novi.TechItEasy.controllers;
 
+import org.apache.coyote.Response;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/api/televisions")
 public class TelevisionController {
+
+    List<String> televisions = new ArrayList<>();
```

Comment:
Je had hier de repository kunnen gebruiken in plaats van deze lijst als mock database. Je had dan ook de Television klasse kunnen gebruiken in de RequestBody en de ResponseBody